### PR TITLE
DOCS-1843 / 21.04 / Changed the setup kubernetes test to use the system static ip.

### PR DESCRIPTION
**Pull request by ericbsd**
Added assert to make sure that the system static ip is present in kubernetes test 01

### CODE REVIEW

File: tests/api2/test_273_kubernetes.py [hunk 1]
```diff
@@ -31,12 +32,12 @@ def test_02_setup_kubernetes(request):
...
         'route_v4_interface': interface,
         'route_v4_gateway': gateway,
-        'node_ip': '0.0.0.0'
+        'node_ip': ip
     }
     results = PUT('/kubernetes/', payload)
     assert results.status_code == 200, results.text
     job_id = results.json()
-    job_status = wait_on_job(job_id, 300)
+    job_status = wait_on_job(job_id, 180)
```
Comment (sonicaj) [inline tests/api2/test_273_kubernetes.py:40]: Let's please keep this to 300

Comment (ericbsd) [reply tests/api2/test_273_kubernetes.py:40]: oh, my master was up-to-date tanks.

Comment (ericbsd) [reply tests/api2/test_273_kubernetes.py]: done

File: tests/api2/test_273_kubernetes.py [hunk 2]
```diff
@@ -20,6 +20,7 @@ def test_01_get_kubernetes_bindip_choices():
     assert results.status_code == 200, results.text
     assert isinstance(results.json(), dict), results.text
     assert results.json()['0.0.0.0'], results.text
+    assert results.json()[ip], results.text
```
Comment (sonicaj) [inline tests/api2/test_273_kubernetes.py:23]: If ip does not exist, we will have a keyerror. I think we should do “ip in results.json()” to check if it exists

Comment (ericbsd) [reply tests/api2/test_273_kubernetes.py:23]: good idea

Comment (ericbsd) [reply tests/api2/test_273_kubernetes.py:23]: I will do the same for 0.0.0.0

Comment (ericbsd) [reply tests/api2/test_273_kubernetes.py]: Done

**Review by sonicaj — APPROVED** [review]
Thanks

**Pull request merged**